invisible_cities/cities/components.py

```diff
@@ -451,6 +453,16 @@ def mcsensors_from_file(paths     : List[str],
                                                        return_raw = False     ,
                                                        db_file    = db_file   ,
                                                        run_no     = run_number)
+        sns_pos_tbl = 'MC/sns_positions'
+        if is_oldformat_file(file_name): sns_pos_tbl = 'MC/sensor_positions'
+
+        nexus_sns_pos       = pd.read_hdf(file_name, sns_pos_tbl)
+        nexus_pmt_ids       = nexus_sns_pos[nexus_sns_pos.sensor_id < 100].sensor_id.values
```

Comment:
This is dangerous and I believe there is another instance in the code where we rely on a magic number. We should write the sensor type in this table in nexus. For now, we can add a comment that clarifies this number.